Registr/Extensions/Date+Extensions.swift

```diff
@@ -24,6 +24,7 @@ extension Date {
         dateFormatter.dateFormat = "dd-MM-yyyy"
         let currentDateFormatted = dateFormatter.string(from: currentDate)
         
+        dateFormatter.locale = Locale(identifier: "da_DK")
```

Comment:
Havde vi brug for denne længere? eller er den lige meget i forhold til den DateFormatter du har lavet?